### ISSUE TITLE
Problems with advert pinned to foot of page for (sighted) keyboard-only users

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/MyopicVicar#2803]
1. For example, after tabbing to the footer, the link to "Terms and conditions" is the last link with automatically visible focus — links further down the page are covered by the advert. Although the subsequent links can be revealed by using the down arrow key, this is not ideal — the actual link that is hidden varies from page to page
2. At the very foot of the page, tabbing takes a user into the advert where there is no visible indication of focus
3. Attempts to go back up the page by using shift+tab are hampered by the ever-changing advert which might have multiple layers of links

As a more general comment, who actually tests the pages for keyboard-only accessibility? You cannot rely on me for testing because my health is failing and I have other, more personally important priorities (such as eating!). Also, I've lost access to ZenHub because the team has increased in size, hitting the limit. Please designate someone to do this basic testing as a matter of course.

I also note that there is no access yet to a skip-link at the very top of the page.

**Comment by AlOneill** [comment]
Another example of the element in focus, this time the "Top of page" link, being hidden underneath the advert:

<img width="1395" alt="Image" src="https://github.com/user-attachments/assets/e70a24d1-5e78-4d3d-ac40-1df102f9c66f" />

**Comment by stsccfr** [comment]
Closing, as I can't reproduce any of this as of now.
